### DISCUSSION TITLE
Multitenant app - How and where to invalidate a session?

### DISCUSSION THREAD

**Discussion by madeinspace** [nextauthjs/next-auth#2854] [Help]
Hi all,

I have a multitenant application and I'm wondering what's the best practice on how to invalidate a session (signout ? ) when the tenant changes?

ie:  domain/tenant1 is authenticated, now if I change to domain/tenant2 I'd like to detect the change and signout tenant1. 

What's best practice, any similar usecases? Thanks!

**Comment by balazsorban44** [comment]
We don't currently support this kind of setup, and I also don't really know how you use NextAuth.js in your project. a related discussion is ongoing in #2414
